code/modules/mob/mob.dm

```diff
@@ -203,6 +203,9 @@ var/next_mob_id = 0
 /mob/proc/restrained()
 	return
 
+/mob/proc/incapacitated()
```

Comment:
Good job on copy-pasta'ing procs that you never override in the human or carbon mobs, but still use in your code, this isn't going to work.